### ISSUE TITLE
Custom user pages

### ISSUE THREAD

**Issue by michaelmnich** [michaelmnich/Sioux#1]
Add Possibility to complicate pages without rebuild all project.

**Comment by michaelmnich** [comment]
Now pages cane be added to project by creating simple web page  based on class **BasePage**.

For example:

```
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Linq;
using System.Net;
using System.Text;
using System.Threading.Tasks;
using ManagingWebSerwer.Conections;
using ManagingWebSerwer.dao;
namespace ManagingWebSerwer.pages.Test
{
    class SamplePageToCompile : BasePage
    {

        public SamplePageToCompile()
        {          
        }

        public override string GetContent()
        {
            return "<br><hr><p style='text-align: center;'>Sample compilated page</p>";
        }


        public override void Set_GET_Params(Dictionary<string, string> Get_params)
        {
            _Get_params = Get_params;
        }

        public override void Set_SET_Params(Dictionary<string, string> Set_params)
        {
            _Set_params = Set_params;
        }

        public override void Set_Cockie_Params(Dictionary<string, Cookie> Cockie_params)
        {
            _Cookie_params = Cockie_params;
        }

        public override string GetUri()
        {
            return "sample"; //This is page url. So to access that page you must type "http://localhost:8080/sample/"
        }
    }
}

```

All pages are store in:  MainSiouxdirecotry/Pages:

For example "C:\Users\admin\Source\Repos\Sioux\Sioux\bin\Debug\pages"


![Bez tytułu](https://user-images.githubusercontent.com/3948281/63641172-23456080-c6aa-11e9-98d5-9fbb5ec15fd5.png)

**Comment by michaelmnich** [comment]
Closed with commit: a3362b39e52c4779992eab8aa4f4df1aab2f0a21
